src/shared/components/TitleArea/TitleArea.style.tsx

```diff
@@ -1,65 +1,77 @@
 import styled from '@emotion/styled'
-import { fluidRange } from 'polished'
 
 import { colors, media, sizes, typography } from '@/shared/theme'
 
 import { Text } from '../Text'
 
-type HelperTextProps = {
-  error?: boolean
-  warning?: boolean
-}
-
-type StyledInputProps = {
-  widthSize: number | null
-}
-
 export const Container = styled.div`
   position: relative;
-  width: fit-content;
-  background-color: ${colors.gray[800]};
 `
 
-export const StyledInput = styled.input<StyledInputProps>`
-  --input-max-width: 60vw;
-  ${media.sm} {
-    --input-max-width: 400px;
-  }
+type StyledInputProps = {
+  touchedAndEmpty?: boolean
+}
 
-  ${media.md} {
-    --input-max-width: 600px;
-  }
+type TitleAreaInfoProps = {
+  error?: boolean
+}
 
-  line-height: 1;
+export const TitleAreaInfo = styled.div<TitleAreaInfoProps>`
+  display: grid;
+  opacity: 0;
+  max-width: 600px;
   padding: ${sizes(1)} 0 ${sizes(2)} ${sizes(2)};
-  ${fluidRange({ prop: 'fontSize', fromSize: '32px', toSize: '40px' })};
+  grid-template-columns: 1fr 1fr;
+  grid-gap: ${sizes(9)};
+`
 
+export const StyledInput = styled.textarea<StyledInputProps>`
+  caret-color: ${colors.blue[500]};
+  line-height: 1;
+  padding: ${sizes(1)} 0 ${sizes(2)} ${sizes(2)};
+  font-size: ${sizes(8)};
   color: white;
   background-color: ${colors.transparent};
-  white-space: nowrap;
-  text-overflow: ellipsis;
-  overflow: hidden;
   font-family: ${typography.fonts.headers};
   font-weight: ${typography.weights.bold};
   border: none;
-  min-width: 100px;
-  max-width: var(--input-max-width);
-  width: ${({ widthSize }) => (widthSize ? `${widthSize}ch` : '100%')};
-  height: ${sizes(13)};
+  width: 100%;
+  resize: none;
+  height: auto;
 
   &:hover {
-    filter: brightness(80%);
+    opacity: 75%;
+  }
+
+  ::placeholder {
+    color: ${colors.gray[500]};
+  }
+
+  + ${TitleAreaInfo} {
+    ${({ touchedAndEmpty }) => touchedAndEmpty && `opacity: 1`};
+  }
+
+  :focus,
+  :invalid {
+    + ${TitleAreaInfo} {
+      opacity: 1;
+    }
+  }
+  ${media.sm} {
+    font-size: ${sizes(12)};
```

Comment:
Let's do what the design does and not specify any queries inside the component. Instead let's a `variant` prop which determines sizing and so on. Also, this seems to big compared to the design